cablelabs/snaps-boot
# Fixes #230

#### What does this PR do?
Automate generic post script execution for servers.

#### Do you have any concerns with this PR?
No

#### How can the reviewer verify this PR?
Deploy snaps_boot and verify that the post script is automatically uploaded and executed.

#### Any background context you want to provide?
Currently specifying post script url for each server is sometimes inconvenient. This PR provides a generic post script file upload and execution mechanism.

#### Screenshots or logs (if appropriate)
#### Questions:
- Have you connected this PR to the issue it resolves?
#230 

- Does the documentation need an update?
Yes, in the PR.

- Does this add new Python dependencies?
No

- Have you added unit or functional tests for this PR?
Yes, added a test to validate post script file execution in CI.

- Does this patch update any configuration files?
No
